src/trans/gpu/external/gpnorm_trans_gpu.F90

```diff
@@ -273,8 +274,8 @@ SUBROUTINE GPNORM_TRANS_GPU(PGP,KFIELDS,KPROMA,PAVE,PMIN,PMAX,LDAVE_ONLY,KRESOL)
   DO JGL=IBEG,IEND
     IGL = D_NPTRLS(MYSETW) + JGL - 1
     DO JF=1,IF_FS
-      ZAVE(JF,JGL)=ZAVE(JF,JGL)*F_RW(IGL)/G_NLOEN(IGL)
-      !write(iunit,*) 'aver inside ',JF,IF_FS,IGL,ZAVE(JF,JGL), F_RW(IGL), G_NLOEN(IGL),ZMINGPN(JF),ZMAXGPN(JF)
+      ZAVE(JF,JGL)=ZAVE(JF,JGL)*F%RW(IGL)/G_NLOEN(IGL)
```

Comment:
Do we need to `COPY(F%RW)` somewhere above?

Reply:
Good point. Let me do it the same way as for the other variables for now and just put F_RW here, add it to the ASSOCIATES and the initial data clause. I think in general this should only be in a present clause because it is set up in setup_trans.